miqa/core/tasks.py

```diff
@@ -25,7 +26,14 @@
 from miqa.learning.nn_classifier import evaluate1
 
 
-def evaluate_data(images: List[Image], session: Session):
+@app.task
+def evaluate_data(image_ids, session_id):
+    try:
+        images = [Image.objects.get(id=image_id) for image_id in image_ids]
+        session = Session.objects.get(id=session_id)
+    except (Image.DoesNotExist, Session.DoesNotExist):
+        raise ValueError('Provide the id of the user and the id of the session.')
```

Comment:
We can remove this `try/except`; the caller should be responsible for validating the IDs being passed in. And if for some reason it doesn't exist, it's preferable just to bubble up the relevant `DoesNotExist` exception.

Also, as a general pattern, if you have two statements that each trigger a certain type of exception, it would be preferable to put them in two separate try/except blocks, one for each of the statement/exception type pairs.